pnas/PNAS-template-main.tex

```diff
@@ -291,20 +288,20 @@ \subsection*{Best response memory-one strategies}
 \(q=(1, 1, 1, 1)\), is a defector \(p^*=(0, 0, 0, 0)\). The strategies
 \(p=(\frac{1}{2}, 0, 0, 0)\) and \(p=(\frac{1}{2}, 0, 0, \frac{1}{2})\) are also
 best responses. The approach taken here is to introduce a compact way of
-constructing the candidate set of all local optimal points, and evaluating which
+constructing the discrete candidate set of all local optimal points, and evaluating which
```

Comment:
`and evaluating the objective function Eq... . This gives the the best response ...`